politeiawww/cmswww.go

```diff
@@ -1047,51 +1047,58 @@ func (p *politeiawww) makeProposalsRequest(method string, route string, v interf
 
 	route = dest + "/api/v1" + route
 
-	// We have to special case post requests since they require to first get
-	// cookies and csrf headers from a Version GET request.
-	if method == http.MethodPost {
-		versionRoute := dest + "/api/v1" + www.RouteVersion
-		req, err := http.NewRequest(http.MethodGet, versionRoute, nil)
+	content := p.memorycache.Get(route + string(requestBody))
+	if content != nil {
+		return content, nil
+	} else {
```

Comment:
Since you're returning in the if statement so you don't need this else statement here.  Try to write code with the least amount of nesting.  The more nested code is the harder it is to read and reason about.